lib/ProductOpener/Display.pm

```diff
@@ -7063,28 +7063,14 @@ sub display_bottom_block($)
 sub display_blocks($)
 {
 	my $request_ref = shift;
-	my $blocks_ref = $request_ref->{blocks_ref};
 
 	my $html = '';
-
-	foreach my $block_ref (@{$blocks_ref}) {
-		$html .= "
-<div class=\"block\">
-<h3 class=\"block_title\">$block_ref->{title}</h3>
-<div class=\"block_content\">
-$block_ref->{content}
-</div>
-</div>
-";
-		if ((defined $block_ref->{id}) and ($block_ref->{id} eq 'my_block')) {
-			$html .= "<!-- end off canvas blocks for small screens -->\n";
-		}
-
-	}
+	my $template_data_ref_blocks->{blocks} = $request_ref->{blocks_ref};
 
 	# Remove empty titles
 	$html =~ s/<div class=\"block_title\"><\/div>//g;
```

Comment:
# Remove empty titles
	$html =~ s/<div class=\"block_title\"><\/div>//g;

--> this can now be removed as you now check for empty titles in the template